spec/javascripts/PaymentGateways/braintree/BraintreeForm.spec.jsx

```diff
@@ -4,9 +4,10 @@ import * as React from 'react'
 
 import { BraintreeForm } from 'PaymentGateways'
 import { mount } from 'enzyme'
+import * as hostedFields from 'braintree-web/hosted-fields'
```

Comment:
This is gonna make the mock fail I think, the import has to happen after calling `jest.mock()`.